### PR TITLE
[acl-loader]: Add --table_name option to update full operation

### PR DESCRIPTION
**Pull request by stcheng**
This option enables the ability to only do the full update within
this table, which means that only this table will be cleared and
reinstalled with the new rules from the JSON file.

Signed-off-by: Shu0T1an ChenG <shuche@microsoft.com>

### CODE REVIEW

**Comment by prsunny** [comment]
@stcheng , can you provide an command output for this change?

File: acl_loader/main.py [hunk 1]
```diff
@@ -356,6 +366,10 @@ def convert_rules(self):
                 warning("%s table does not exist" % (table_name))
                 continue
 
+            if self.current_table is not None and self.current_table != table_name
+                warning("%s table does not match the current table %s" % (table_name, current_table))
```
Comment (prsunny) [inline acl_loader/main.py]: self.current_table in warning?

File: acl_loader/main.py [hunk 2]
```diff
@@ -370,11 +384,14 @@ def convert_rules(self):
...
+        the current_table is not empty, only rules within that table will
+        be removed and new rules in that table will be installed.
         :return:
         """
         for key in self.rules_db_info.keys():
-            self.configdb.mod_entry(self.ACL_RULE, key, None)
+            if self.current_table is None or self.current_table == key[0]:
+               self.configdb.mod_entry(self.ACL_RULE, key, None)
 
         self.configdb.mod_config({self.ACL_RULE: self.rules_info})
```
[the remark this answers is not in the document]
Comment (lguohan) [reply acl_loader/main.py]: in this case, the rules in other rules should be ignored.

[the remark this answers is not in the document]
Comment (stcheng) [reply acl_loader/main.py]: okay i could remove the warning.

File: acl_loader/main.py (hunk 1 above)
Comment (stcheng) [reply acl_loader/main.py]: ah yes. but just removed this line.

**Comment by stcheng** [comment]
sample command:
`acl-loader update full x.json --table_name=EVERFLOW  --session_name=session_name`

**Pull request merged**